### PR TITLE
switch to lodash, remove underscore mixin

### PR DESCRIPTION
**Pull request by wraithgar**

### CODE REVIEW

**Comment by wraithgar** [comment]
Closes #6

File: ampersand-filtered-subcollection.js [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
+var includes = require('lodash.includes');
+var difference = require('lodash.difference');
+var forEach = require('lodash.foreach');
+var every = require('lodash.every');
+var assign = require('lodash.assign');
+var isArray = require('lodash.isarray');
+var isEqual = require('lodash.isequal');
+var keys = require('lodash.keys');
+var reduce = require('lodash.reduce');
+var sortBy = require('lodash.sortBy');
```
Comment (lukekarrys) [inline ampersand-filtered-subcollection.js]: Should be `lodash.sortby` I think.

Comment (wraithgar) [inline ampersand-filtered-subcollection.js]: You're right

**Comment by lukekarrys** [comment]
+1

**Comment by wraithgar** [comment]
readme updated to match #12

**Comment by klaemo** [comment]
LGTM

**Comment by wraithgar** [comment]
published as `v2.0.0`

**Pull request merged**